zim/gui/pageview/textbuffer.py

```diff
@@ -796,8 +797,9 @@ def _create_link_tag(self, text, href, **attrib):
 		if hasattr(href, 'uri'):
 			href = href.uri
 		assert isinstance(href, str) or href is None
-
-		tag = self.create_tag(None, **self.tag_styles['link'])
+		is_internal_link = (link_type(href or '') == 'page')
+		styles = self.tag_styles['internal-link' if is_internal_link else 'link']
```

Comment:
Nit: `is_internal_link` is good for readability, but `styles` only exists to get unpacked and doesn't communicate extra info compared to `self.tag_styles`